services/ecr/apis/v1alpha1/enums.go

```diff
@@ -48,7 +41,6 @@ const (
 	ImageFailureCode_ImageNotFound                 ImageFailureCode = "ImageNotFound"
 	ImageFailureCode_MissingDigestAndTag           ImageFailureCode = "MissingDigestAndTag"
 	ImageFailureCode_ImageReferencedByManifestList ImageFailureCode = "ImageReferencedByManifestList"
-	ImageFailureCode_KmsError                      ImageFailureCode = "KmsError"
```

Comment:
Please revert these changes. They were due to you running `make build-controller SERVICE=ecr` with an older version of `aws-sdk-go` :)

Reply:
The `make build-controller SERVICE=ecr` was producing this even after clearing go mod caches.

I find that setting `ACK_GENERATE_CACHE_DIR` environment variable to a value resulted in resolving this issue.
* The default value `~/.cache/aws-controllers-k8s` (set inside `build-controller.sh`) did not create any `.cache` directory inside user `home`.

latest PR includes files generated using 
```
make build-controller SERVICE=elasticache && make build-controller SERVICE=sns && make build-controller SERVICE=ecr && make build-controller SERVICE=s3
```